templates/yeet/scripts/yeet_pkg.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+exit_success=0
+exit_failure=1
+mode="pkg"
```

Comment:
lb above

Reply:
or group the params in one paragraph consts in other.

mode is just a param thats defaulted